src/cli/backup/backup.go

```diff
@@ -187,7 +187,7 @@ func genericCreateCommand(
 
 		bo, err := r.NewBackupWithLookup(ictx, discSel, ins)
 		if err != nil {
-			errs = append(errs, clues.Wrap(err, owner).WithClues(ictx))
+			errs = append(errs, clues.WrapWC(ictx, err, owner))
```

Comment:
do you happen to know if it's possible to make a lint rule for this? It seems like one of those things that will be difficult to get everyone on the same page for without any instances of the old form getting committed

It's been a while since I added something to forbidigo, but I think it would only match on either `Wrap`/`Stack` or `WithClues`, but not both. I'm not sure what [tree-sitter](https://tree-sitter.github.io/tree-sitter/) (used for trailing `)`) would be able to match on. Maybe both since iirc it did seem to support a fairly complex grammar

Reply:
I'm a little hesitant about adding a linter here, since the rule is not `"only ever use WrapWC"`.  The rule is, actually, `"don't use WithClues, use FooWC(ctx, ...)`.  Though, now that I think about it, that's an easier linter anyway...